benchmark/README.md

```diff
@@ -13,6 +13,58 @@ The minimum sample size is set to 10 to perform statistical analysis on benchmar
 
 ## Results
 
+### `v0.5.0` release
+
+```
+Beginning NoopTracerProvider Benchmark...
+  5 tests completed.
+
+  #startSpan                     x 766,888,261 ops/sec ±1.81% (20 runs sampled)
+  #startSpan:parent              x 755,332,363 ops/sec ±1.51% (20 runs sampled)
+  #startSpan with attribute      x 765,932,668 ops/sec ±1.45% (20 runs sampled)
+  #startSpan with 30 attributes  x   1,333,216 ops/sec ±10.66% (20 runs sampled)
+  #startSpan with 100 attributes x     477,974 ops/sec ±3.66% (20 runs sampled)
+
+Beginning BasicTracerProvider Benchmark...
+  5 tests completed.
+
+  #startSpan                     x 94,710 ops/sec ±19.20% (20 runs sampled)
+  #startSpan:parent              x 62,938 ops/sec ±3.77% (20 runs sampled)
+  #startSpan with attribute      x 93,389 ops/sec ±7.70% (20 runs sampled)
+  #startSpan with 30 attributes  x 33,753 ops/sec ±8.07% (20 runs sampled)
+  #startSpan with 100 attributes x  2,497 ops/sec ±14.78% (20 runs sampled)
+
+Beginning BasicTracerProvider with SimpleSpanProcessor Benchmark...
+  5 tests completed.
+
+  #startSpan                     x 100,159 ops/sec ±7.17% (20 runs sampled)
+  #startSpan:parent              x  63,848 ops/sec ±5.78% (20 runs sampled)
+  #startSpan with attribute      x  96,301 ops/sec ±9.39% (20 runs sampled)
+  #startSpan with 30 attributes  x  36,410 ops/sec ±2.21% (20 runs sampled)
+  #startSpan with 100 attributes x   3,549 ops/sec ±3.33% (20 runs sampled)
+
+Beginning BasicTracerProvider with BatchSpanProcessor Benchmark...
+  5 tests completed.
+
+  #startSpan                     x  90,992 ops/sec ±17.91% (20 runs sampled)
+  #startSpan:parent              x  64,590 ops/sec ±4.18% (20 runs sampled)
+  #startSpan with attribute      x 107,706 ops/sec ±2.21% (20 runs sampled)
+  #startSpan with 30 attributes  x  24,199 ops/sec ±45.57% (20 runs sampled)
+  #startSpan with 100 attributes x   2,645 ops/sec ±9.86% (20 runs sampled)
+
+Beginning B3Propagator Benchmark...
+  2 tests completed.
+
+  #Inject  x 2,018,725 ops/sec ±3.49% (100 runs sampled)
```

Comment:
Both the propagators throughput (ops/sec) has dropped significantly since `0.3.3` version.

Reply:
I believe this is most likely due to the getter/setter being functions instead of property accesses now. Because we are still able to process multiple millions of operations per second, I believe this is unlikely to be an issue or bottleneck in any real world js applications.